public/style.css

```diff
@@ -28,22 +28,26 @@ h3 {
   max-width: 700px;
   margin: 15px auto;
 }
+
```

Comment:
Be consistent with your spacing. Either add one space after every class, or don't 💃